pages.de/common/nix-build.md

```diff
@@ -0,0 +1,12 @@
+# nix-build
+
+> Erstellen eines Nix-Ausdruck.
+> Weitere Informationen: <https://nixos.org/releases/nix/latest/manual#sec-nix-build>.
+
+- Erstellen eines Nix-Ausdrucks:
+
+`nix-build --attr {{ausdruck_name}}`
+
+- Erstellen eines Nix-Ausdruck mit Sandbox (auf nicht-NixOS):
+
+`nix-build --attr {{ausdruck_name}} --option sandbox true`
```

Comment:
The following fix could be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/nix-build.md) page is updated:

> Concrete `ausdruck_name` value should be provided.

It's better to use certain values.